packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -129,8 +129,7 @@ function filterRuntimeOptionsForVersion(
 		options = {
 			compressionOptions,
 			enableGroupedBatching,
-			// need to investigate - some small number of t9s tests time out with this option on.
-			// chunkSizeInBytes,
+			chunkSizeInBytes,
 			enableRuntimeIdCompressor,
```

Comment:
@andre4i - FYI, worth taking a look, maybe you will have theory. Comment based on using chunkSizeInBytes = 200 (I'm planning to try higher limit).

Results:
https://dev.azure.com/fluidframework/public/_build/results?buildId=248612&view=ms.vss-test-web.build-test-results-tab
12 test fail with timeouts, only for t9s (not local). 
I can somewhat repro it locally - I usually have 1-3 test failing.

The only theory I have - it creates too many ops, maybe, and thus test does not have enough time to complete whatever it is doing? And T9s tests being much slower (and slower than my local machine), possibly not having enough time, even despite higher limit (I think it's 5 seconds vs 2 seconds in local runs?)

The part that I can't explain - that was passing before RC2 was cut off. No clue what is the relationship (or of there were other changes in main that caused it).

Reply:
latest iteration "fixes" this problem. Not sure if it's good direction, but it's a direction :)

Reply:
If it passes for all other drivers, then It's a safe assumption the failures are about T9s and not the tests or the things the tests are covering. A large number of ops will likely topple it and cause timeouts. Another way of verifying it is to disable some of the failing tests and see if other tests start failing. Can we get the T9s logs for the run? I recall there was a way of doing so but I can't seem to find them anymore.

Reply:
No, it's usually same tests fail - I run so many builds that I have good statistic :)

Reply:
In the past, we had tests which were deemed as being broken in T9s for various reasons (but mostly timeouts which 'smelled' like race conditions). The mitigation was usually to disable the test. But when the test was disabled, the next test (or some tests which was supposed to run after the disabled test) failed the exact same way. So, it was something along the lines of a new test being added, or some new behavior which would now topple T9s in a consistent manner, making seem like the tests were the problem.